src/WebViewTypes.ts

```diff
@@ -251,7 +247,7 @@ export interface WebViewNativeConfig {
   component?:
     | typeof NativeWebViewIOS
     | typeof NativeWebViewMacOS
-    | typeof NativeWebViewAndroid;
+    | typeof NativeWebViewComponent;
```

Comment:
Won't Fabric force normalization to a single interface? Are you keeping iOS and MacOS until it's fully migrated?